docs/sphinx/using/simulators.rst

```diff
@@ -90,3 +90,70 @@ options to :code:`nvq++`
 .. code:: bash 
 
     nvq++ --target tensornet src.cpp ...
+
+GPU Simulation Requirements
+==================================
+
+For programmers using the CUDA Quantum Python wheels, extra configuration is needed to install
+our GPU enabled simulation backends.
+
+System Level Dependencies
+++++++++++++++++++++++++++
+
+The system level dependencies for these backends include:
+* An NVIDIA GPU with compute capability 7.0+
+* Driver: Linux (450.80.02+ for CUDA 11, 525.60.13+ for CUDA 12)
+* CUDA Toolkit 11.x or newer
+* Python 3.9+
```

Comment:
Should this be a link to the tables in https://github.com/NVIDIA/cuda-quantum/pull/707 ? Should the additional information here be added to those tables?

Reply:
I've removed this and am linking to the table now. I've also added the python version to the "supported systems" section